onnxscript/tensor.py

```diff
@@ -39,6 +39,10 @@ def value(self) -> np.ndarray:
     def rank(self) -> int:
         return len(self.value.shape)
```

Comment:
One issue when I was using rank was np.ndarray does not have it, and so calling it in a trace only function will fail because it was not converted to a Tensor. Now that we have a special class for these functions (TracedOnnxFunction) we may be able to wrap ndarrays there? Or maybe constraint users to always provide Tensors instead of ndarrays.

Reply:
Yes, we could do something like that for tracing-mode. Orthogonal to current PR, I guess ... (at this point) this code is not meant for tracing-mode.